src/index.html

```diff
@@ -1,12 +1,448 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/styles.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__top">
+        <a class ="header__logo" href="#">
+          <img
+            class="header__logo"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                class="nav__link
+                is-active"
+                href="#"
+              >
+              apple
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+              samsung
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+              smartphones
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                data-qa="nav-hover"
+                href="#"
+              >
+              laptops & computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+              gadgets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+              tablets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+              photo
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+              video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <div class="product-card" data-qa="card">
```

Comment:
You can use more semantic tag here.

```suggestion
      <article class="product-card" data-qa="card">
```